docs/source/tutorial/understanding_evaluation.rst

```diff
@@ -180,6 +180,51 @@ may be additional known triples :math:`(h, r, t')` for :math:`t \neq t'`. If the
 filtered evaluation setting ignores for a given triple :math:`(h, r, t)` the scores of all other *known* true triples
```

Comment:
this is the first time a new use will be reading about this filtering in the docs, assuming they're going from top to bottom, so the first paragraph also needs to be improved